datalake/scripts/cli.py

```diff
@@ -209,6 +209,9 @@ def _uploader(**kwargs):
 
 
 def _ms_to_iso(ms):
+    if not ms:
+        # Return None if time is null
+        return
```

Comment:
I'd rather do this up a level or two where the semantic meaning of `None` in this context is more obvious.